sql/moz-fx-data-shared-prod/fenix_derived/clients_yearly_v1/init.sql

```diff
@@ -1,18 +1,50 @@
-CREATE TABLE IF NOT EXISTS
+CREATE OR REPLACE TABLE
   fenix_derived.clients_yearly_v1
 PARTITION BY
-  submission_date
+  (submission_date)
 CLUSTER BY
-  sample_id,
-  client_id
+  (sample_id)
 OPTIONS
   (require_partition_filter = TRUE)
 AS
+WITH base AS (
+  -- There are duplicates now in `baseline_clients_daily` because of the join with `clients_first_seen`,
+  -- so we take the minimum
+  SELECT
+    * EXCEPT (normalized_app_id),
+    ROW_NUMBER() OVER (
+      PARTITION BY
+        client_id,
+        submission_date
+      ORDER BY
+        first_seen_date ASC,
+        first_run_date ASC
+    ) AS rn,
+  FROM
+    fenix.baseline_clients_daily
+  WHERE
+    submission_date >= "2021-08-01"
```

Comment:
Is it possible to just read from the last 365 `submission_date`'s here? I think in production the query drops rows older than that
```suggestion
    submission_date >= DATE_SUB(CURRENT_DATE(), INTERVAL 365 DAY)
```